laurent22/joplin
# Mobile: Resolves#7675 Fix ViewPropType error in react-native-camera

- This change includes patch-package as dependency to apply the patch to react-native-camera package. The patch imports ViewPropType from deprecated-react-native-prop-types instead of directly importing from react-native
- More context can be found at #7675


Testing instructions:
 - Follow the instructions [here](https://github.com/laurent22/joplin/blob/dev/BUILD.md#testing-the-mobile-application) to build and start the mobile app
 - Once app shows up in the device/emulator, create a new note
 - From the top right menu option --> Attach
 - Confirm that camera opens up
